.github/workflows/ci.yml

```diff
@@ -525,7 +525,7 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: |
-            ghcr.io/fuellabs/forc
+            ghcr.io/fuellabs/sway
```

Comment:
oh interesting, I assumed the ghrc registry was tied to the repo name. Guess this is a non-issue.

Reply:
It is an issue since only one of them was changed to `forc` in #1543! Could you do a follow-up PR to change them _both_ to `forc`, and we can bikeshed in that issue if the ghcr registry should be `forc` vs `sway`? (I'm actually leaning towards `forc`).